test/test_cyclic_socketcan.py

```diff
@@ -244,27 +244,6 @@ def test_cyclic_initializer_different_arbitration_ids(self):
         with self.assertRaises(ValueError):
             task = self._send_bus.send_periodic(messages, self.PERIOD)
 
-    def test_create_same_id_raises_exception(self):
-        messages_a = can.Message(
-            arbitration_id=0x401,
-            data=[0x11, 0x11, 0x11, 0x11, 0x11, 0x11],
-            is_extended_id=False,
-        )
-
-        messages_b = can.Message(
-            arbitration_id=0x401,
-            data=[0x22, 0x22, 0x22, 0x22, 0x22, 0x22],
-            is_extended_id=False,
-        )
-
-        task_a = self._send_bus.send_periodic(messages_a, 1)
-        self.assertIsInstance(task_a, can.broadcastmanager.CyclicSendTaskABC)
-
-        # The second one raises a ValueError when we attempt to create a new
-        # Task, since it has the same arbitration ID.
-        with self.assertRaises(ValueError):
```

Comment:
I think this test could probably be repurposed to ensure that we receive both messages, rather than throwing an exception.

Reply:
Modified this test.